src/runtime/capabilities.ts

```diff
@@ -303,6 +310,14 @@ export abstract class BooleanCapability extends Capability {
     return true;
   }
 
+  /**
+   * Returns true, if the value is false and the given capability is null.
```

Comment:
If the policy specifies `encryption: false`, but in the recipe handle nothing about encryption is declared, then the Encryption capability in a policy will be null.
I tweaked this method to always return true, if value is false.
================================
On a more general note:
In a follow up PR, I'm planning to add `unspecified` value to all Capability classes and have Capabilities contain ranges for all known Capability types.
This is because `unspecified` requires different behavior in different use cases (and per Capability implementation), for example:
- when running CapabilitiesResolver, if Persistence wasn't specified for a handle, then any StorageDriver (whether persistent or not) is fine. Same for Encryption.
- however when verifying against a Policy, if Persistence isn't specified, this is an error (no matter if Policy says Ram or Disk, the handle's Capability must be explicit). But not so for Encryption - if policy has `false` for encrypted, handle's Capability might not specify anything about it.

I might add a section on this in one of the docs as well, but wanted to first articulate it here, so make sure it made sense. WDYT?

Reply:
That sounds reasonable to me. Make sure you add copious comments, it's a pretty subtle concept..